### PR TITLE
DO NOT MERGE: 7.2.0-preview.2 changes

### PR DESCRIPTION
**Pull request by rjmholt**
# PR Summary

Changes made for the 7.2.0-preview.2 release.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Comment by rjmholt** [comment]
Once this is approved I'll need to create a merge commit manually

File: tools/releaseBuild/azureDevOps/templates/release-ReleaseToNuGet.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ steps:
     $releaseVersion = Get-Content "$ENV:PIPELINE_WORKSPACE/releasePipeline/metadata/release.json" | ConvertFrom-Json | Select-Object -ExpandProperty 'ReleaseVersion'
     $globalToolPath = "$ENV:PIPELINE_WORKSPACE/releasePipeline/finalResults/PowerShell.$releaseVersion.nupkg"
     ### -WhatIf to make sure we do not release global tool. Remove -WhatIf when the PowerShell name reservation is done.
-    Copy-Item $globalToolPath -Destination "$(Pipeline.Workspace)/release" -WhatIf
+    ### Copy-Item $globalToolPath -Destination "$(Pipeline.Workspace)/release" -WhatIf
```
Comment (daxian-dbw) [inline tools/releaseBuild/azureDevOps/templates/release-ReleaseToNuGet.yml:33]: Shall this change be taken back to GH master?

**Review by daxian-dbw — COMMENTED** [review]
For changes except for the changelog, do we need to take them back?

File: tools/releaseBuild/azureDevOps/templates/release-ReleaseToNuGet.yml (hunk 1 above)
Comment (rjmholt) [reply tools/releaseBuild/azureDevOps/templates/release-ReleaseToNuGet.yml:33]: Seems like it shouldn't be -- I'll remove it

**Comment by rjmholt** [comment]
Closed in favour of https://github.com/PowerShell/PowerShell/pull/14434

**Pull request closed without merging**